src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'books/books/ADD_BOOK';
+const REMOVE_BOOK = 'books/books/REMOVE_BOOK';
+
+export const addBook = (book) => ({
+  type: ADD_BOOK,
+  payload: book,
+});
+
+export const removeBook = (book) => ({
+  type: REMOVE_BOOK,
+  payload: book,
+});
+
+const books = [];
+
+const booksReducer = (state = books, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+    case REMOVE_BOOK:
+      return [...state, action.payload];
```

Comment:
- [ ] Kindly use the function ES6 filter() method to create a new array, which will not contain the book you want to remove from the store (filter by the id key - i.e.: return state.filter(book => book.id !== id). The one you have is currently returning the whole data and nothing is deleted.